api/src/controllers/venue.ts

```diff
@@ -34,39 +36,33 @@ const venuesController = (clients: Clients, config: Config): EventsHandler => {
 
     const filters = pickFiltersFromQuery(["title", "id"], params, venuesFilter);
 
-    try {
```

Comment:
This flagged try/catch as redundant, since it only throws the error it would throw anyway